frontend/src/constants/Routes.ts

```diff
@@ -13,3 +13,9 @@ export const CREATE_ENTITY_PAGE = "/entity/create";
 export const UPDATE_ENTITY_PAGE = "/entity/update";
 
 export const HOOKS_PAGE = "/hooks";
+
+export const COURSE_OVERVIEW_DASHBOARD_BASE_ROUTE = "/course"
```

Comment:
This route would be for the learner's side correct? Not sure if we need this right now, but I'm down to leave it in.

Reply:
This would be for the list of courses a user is enrolled in, but it seems that page is archived now. It still seems like it would be a requirement so I think we can leave it in?